network/p2p/scoring/scoring_test.go

```diff
@@ -136,7 +136,7 @@ func TestInvalidCtrlMsgScoringIntegration(t *testing.T) {
 	})
 
 	// now simulates node2 spamming node1 with invalid gossipsub control messages.
-	for i := 0; i < 30; i++ {
+	for i := 0; i < 750; i++ {
```

Comment:
Now that decay will start lower than .99 and only be incremented by default .01 we need to spam more messages in this test so that the node gets disallow listed.